### PR TITLE
Add carousel and update hero sections

### LINKED ISSUE #287: Add carousel
Please add the following banners as images

Mockup: 
image
image

Assets:
Carousel1
Carousel1
CarouselBox

### PR DESCRIPTION
**Pull request by mativanarquero**
Closes #287 

c_s
m_c_s

### CODE REVIEW

File: src/pages/SparkSwapWebsite/components/Hero/Hero.tsx [hunk 1]
```diff
@@ -276,49 +274,91 @@ const Website: React.FC = () => {
...
-  return (
-    <PageSection direction="column" id="hero">
-      <BG>
-        <HomeWrapper>
-          <StyledFlex>
-            {/* <HeadingContainer>
-              <img src="images/Website/sparkswap-heading.png" alt="SparkSwap Heading" />
-            </HeadingContainer> */}
-            <CustomHeading>New and Improved <br/>
+  const SparkDeFiNowSwap = () => {
```
Comment (medardm) [inline src/pages/SparkSwapWebsite/components/Hero/Hero.tsx:277]: padouble check kaini @kojiadrianojr po

**Review by medardm — APPROVED** [review]
LGTM

File: src/pages/SparkSwapWebsite/components/Hero/Hero.tsx [hunk 2]
```diff
@@ -276,49 +274,91 @@ const Website: React.FC = () => {
...
+          <CustomHeading>A New and Improved <br/>
+            <StyledSpan>Spark</StyledSpan>
+            <StyledSpan style={{ color: '#0071BC' }}>Swap</StyledSpan>
+            <StyledSpan style={{ color: '#FFFFFF' }}>{' '}DEX</StyledSpan> <br/> is coming!
+          </CustomHeading>
+          <SubHeaderText>
+            An improved UI+UX, upgraded Info Page, <br/>
+            Sparky VIP Club perks integration, <br/>
+            Liquidity Lockers, Limit Order, and <br/>
+            perpetial, all coming this 2022! <br/>
```
Comment (swengr-janan) [inline src/pages/SparkSwapWebsite/components/Hero/Hero.tsx:325]: change to **_perpetual_**

**Review by swengr-janan — CHANGES_REQUESTED** [review]

**Comment by swengr-janan** [comment]
Improve ta din ining carousel bullets @mativanarquero 

Mockup:
Screen Shot 2022-06-30 at 11 35 06 AM

Current: 
Screen Shot 2022-06-30 at 11 35 29 AM

**Comment by mativanarquero** [comment]
> Improve ta din ining carousel bullets @mativanarquero
> 
> Mockup: Screen Shot 2022-06-30 at 11 35 06 AM
> 
> Current: Screen Shot 2022-06-30 at 11 35 29 AM

PR updated
c_u

**Comment by swengr-janan** [comment]
@mativanarquero bakit 4 na items to? Dapat 2 lang kasi duplicate na yung iba
Screen Shot 2022-07-01 at 5 25 41 PM

**Comment by swengr-janan** [comment]
@mativanarquero bakit 4 na items to? Dapat 2 lang kasi duplicate na yung iba
Screen Shot 2022-07-01 at 5 25 41 PM

**Comment by mativanarquero** [comment]
> @mativanarquero bakit 4 na items to? Dapat 2 lang kasi duplicate na yung iba Screen Shot 2022-07-01 at 5 25 41 PM

PR updated

**Review by swengr-janan — APPROVED** [review]
AGITH

**Pull request merged**